packages/notification-service/src/firebase.ts

```diff
@@ -115,6 +116,14 @@ export function initializeDb() {
       console.error('Rewards senders data read failed:', errorObject.code)
     }
   )
+
+  database.ref('/inviteRewardAddresses').on('value', (snapshot) => {
+    const addresses = (snapshot && snapshot.val()) ?? []
```

Comment:
nit

did you mean to use null coalescing here?  I find it unexpected because the left-hand expression uses that logical and so i'd expect a boolean as a result (but i guess that isn't necessarily the case!) which would mean the null coalesce always returns the left-hand .  If `snapshot` could be `undefined` and `.val()` returns `null`, `undefined`, or an array, I find this easier to grok:

```
snapshot.?val() ?? []
```

Reply:
I agree, victim of copy-pasting :P